cwiczenie3/osoba.php

```diff
@@ -38,27 +39,26 @@ public function roznicaWieku(Osoba $osoba)
     {
         $Y1 = $osoba->rok;
         $Y2 = $this->rok;
-        $wiek1= new DateTime($Y1);
+        $wiek1 = new DateTime($Y1);
         $wiek2 = new DateTime($Y2);
         $roznica = $wiek1->diff($wiek2);
 
-        switch ($roznica->format('%y'))
-        {
-            case (1);
-                return "Roznica wieku miedzy przyjaciolmi wynosi " . $roznica->format('%y') . " rok.";
-                break;
 
-            case (2) || (3) || (4);
-                return "Roznica wieku miedzy przyjaciolmi wynosi " . $roznica->format('%y') . " lata.";
-                break;
-        }
-        if (($roznica->format('%y'))>(5))
-        {
-            return "Roznica wieku miedzy przyjaciolmi wynosi " . $roznica->format('%y') . " lat.";
-        }
+        return "Roznica wieku miedzy przyjaciolmi wynosi " . $roznica->format('%y') . " lat.";
+    }
 
 
+    public function setTransport($rodzajtransportu)
```

Comment:
Dodaj type hint

Reply:
Inaczej ejstem w stanie wykonać `->setTransport(1234)`. 1234 nie jest transportem.

Reply:
o takie coś chodzi:
public function setTransport(string $rodzajtransportu)?

storm wywala błąd, że możliwe to jest od wersji php7
Sprawdzałem localhost ma 7.2
W przeglądarce nic się nie zmieniło, a mam wpisane właśnie 1234....

Reply:
Nie, o to: `setTransport(Pojazdy $rodzajTransportu)`